src/ngraph/runtime/gpu/gpu_call_frame.cpp

```diff
@@ -61,15 +61,15 @@ void runtime::gpu::GPU_CallFrame::call(
         outputs.push_back(tv->m_allocated_buffer_pool);
     }
 
-    m_compiled_function(inputs.data(), outputs.data(), m_external_function->m_ctx.get());
+    m_compiled_function(inputs.data(), outputs.data(), m_external_function->ctx().get());
```

Comment:
The change is good. My question is are we still using `gpu_call_frame`? Is it in `gpu_backend' now? if so, shall we delete this file?